### ISSUE TITLE
TS2322: Type '{ children: Element; }' is not assignable to type 'IntrinsicAttributes & RecoilRootProps'.

### ISSUE THREAD

**Issue by Hanggi** [facebookexperimental/Recoil#1726]
The env is React 18 with typescript.

I trying to add <RecoilRoot> to the root of React application.

But it shows:

```
TS2322: Type '{ children: Element; }' is not assignable to type 'IntrinsicAttributes & RecoilRootProps'.
```

**Comment by Hanggi** [comment]
It seems about to https://github.com/facebookexperimental/Recoil/pull/1718

**Comment by suren-atoyan** [comment]
@drarmstr any updates on this? ☝️

**Comment by mondaychen** [comment]
We'll try to merge https://github.com/facebookexperimental/Recoil/pull/1718 today and cut a new release ASAP

**Comment by drarmstr** [comment]
Duplicate of #1717

**Comment by drarmstr** [comment]
Published 0.7.1
